pyproject.toml

```diff
@@ -15,7 +15,8 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
 ]
 dependencies = [
-     "GTFSrDB @ git+https://github.com/public-transport/gtfsrdb"
+     "GTFSrDB @ git+https://github.com/public-transport/gtfsrdb",
+     "pandas"
```

Comment:
I would suggest to specify pandas 2.X to avoid issues due to accidentally installing pandas 1.X